app/scripts/services/mobileClients.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+angular.module("openshiftConsole")
+  .factory("MobileClientsService", function(DataService) {
+
+    var mobileclientVersion = {
```

Comment:
move to constants

Reply:
I believe @witmicko has this value in constants in his changes?